silkworm/silkrpc/core/evm_executor.hpp

```diff
@@ -114,4 +109,25 @@ class EVMExecutor {
     protocol::RuleSetPtr rule_set_;
 };
 
+struct ExecutionResult {
+    std::optional<int64_t> error_code{std::nullopt};
```

Comment:
You can simplify this: `std::optional<int64_t> error_code;`

Reply:
ok